ui/page/transaction/transactions_page.go

```diff
@@ -186,11 +193,12 @@ func (pg *TransactionsPage) refreshAvailableTxType() {
 		}
 		pg.txTypeDropDown = pg.Theme.DropDown(items, values.TxDropdownGroup, 2)
 		pg.ParentWindow().Reload()
+		pg.showLoader = false
```

Comment:
How long does it take for this goroutine to complete? Can we do this in the main thread instead of in the background and then showing a loader until it completes?